scripts/create-agent.ts

```diff
@@ -95,7 +95,7 @@ class VargasJRAgentCreator {
       });
       
       if (result.KeyMaterial) {
-        const sshDir = `${process.env.HOME}/.ssh`;
+        const sshDir = `${process.env.HOME || '/home/ubuntu'}/.ssh`;
```

Comment:
In all of these other places where we try to use HOME, use tmpdir as well to make it consistent